packages/aws-cdk/lib/cli.ts

```diff
@@ -38,13 +41,21 @@ if (!process.stdout.isTTY) {
 }
 
 export async function exec(args: string[], synthesizer?: Synthesizer): Promise<number | void> {
-  const argv = await parseCommandLineArguments(args);
+  const argv: CliArguments = convertYargsToCliArgs(await parseCommandLineArguments(args));
+
+  debug('Command line arguments:', argv);
+
+  const configuration = new Configuration({
+    commandLineArguments: argv,
+  });
+  await configuration.load();
+  const settings = configuration.settings.all;
```

Comment:
Side note: With that we still have the weird duplicity of having `Configuration` and `Settings`

Reply:
we're passing `Configuration` to too many places right now for it to make sense for me to change this in this PR. Agree that that's the correct end state.